DESCRIPTION

```diff
@@ -17,7 +17,8 @@ Authors@R: c(
 	  person(family = "PSI special interest group Application and Implementation of Methodologies in Statistics", role = c("cph", "fnd")),
     person(family = "R Validation Hub", role = c("cph", "fnd")),
     person(family = "GSK contributors", role = "ctb", comment = "Author of 'pharmaR/pkg_explorer' repo"),
-    person(family = "Appsilon", role = "ctb")
+    person(family = "Appsilon", role = "ctb"),
+    person("Narayanan","Iyer",role = "ctb", email = "NarayananSeshan.Iyer@pfizer.com")
```

Comment:
I would add your item before the "family" group of contributors since you are a person and not an organization.